obspy/io/resp/nrl.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+nrl.py for navigating, and picking responses from the
+Nominal Response Library.
+
+:copyright:
+    Lloyd Carothers IRIS/PASSCAL, 2016
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+'''
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+# Urllib for py 2 and 3
+from future.standard_library import install_aliases
+install_aliases()  # NOQA
+from urllib.parse import urljoin
+from urllib.request import urlopen
+from os.path import join, dirname
+from ConfigParser import SafeConfigParser
+
+
+class NRL:
+    """
+    Object representing the Nominal Response library.
+    Can be Accessed online from the DMC or with a local copy
+    Decision tree: Usually!!!! depth of tree and order can change!
+    Sensor, make, model, period, sensitivity
+    Datalogger, make, model, gain, sample rate
+
+    The NRL has txt desision files formatted as windows ini
+
+    """
```

Comment:
The NRL docstring should contain the parameter/type description section for `NRL.__init__()`, so that it is visible to users when looking up the class.